.github/workflows/api.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy to Heroku (api)
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "dist/apps/api/**"
+      - ".github/workflows/api.yml"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```

Comment:
We should never run on latest versions as when they update they can make things go sour.